Tasks/DownloadPackageV0/download.ts

```diff
@@ -13,7 +13,7 @@ import { WebApi } from 'azure-devops-node-api';
 tl.setResourcePath(path.join(__dirname, 'task.json'));
 
 async function main(): Promise<void> {
-	tl.warning("This task will be deprecated soon. Please switch to using DownloadPackage@1");
+	tl.warning("This task is being deprecated. All builds will break by 11-27-2023. Please switch to using DownloadPackage@1 as soon as possible.");
```

Comment:
For clarity, consider `"This task is deprecated. Builds that use it will break on 11-27-2023. Please switch to using DownloadPackage@1 as soon as possible."`

Reply:
The terminology is consistent with the public documentation: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/download-package-v0?view=azure-pipelines